spec/device_detector/version_extractor_spec.rb

```diff
@@ -36,7 +37,12 @@
       end
 
       it 'returns the correct version' do
-        expect(extractor.call).to eq('BonEcho (2.0)')
+        expect(extractor.call).to eq(version)
+      end
+
+      it 'should remove trailing white spaces' do
+        regex_meta['version'] = regex_meta['version'] + '   ' # 3 trailing spaces
```

Comment:
You could assign this line to a variable called `version_with_trailing_spaces`, and than you wouldn't need the comment ;) This makes for better maintenance of the code, as the variable name is unlikely to get out of sync, as the comment does.